randovania/data/json_data/prime3.txt

```diff
@@ -1087,45 +1166,65 @@ Asset id: 8374398449894515797
   > Dock to Hidden Court
       Trivial
 
+> Event - Falls of Fire Wall; Heals? False
+  * Event Falls of Fire Wall
+  > Dock to G.F.S. Theseus
+      Trivial
+
 ----------------
 Hillside Vista
 Asset id: 13902157786380317936
 > Dock to Grand Court; Heals? False
   * Normal Door to Grand Court/Dock to Hillside Vista
   > Dock to Reliquary I
-      Trivial
+      Grapple Lasso
   > Pickup (Missile Expansion)
       Trivial
+  > Event - Statue
+      Grapple Lasso
 
 > Dock to Reliquary I; Heals? False
   * Normal Door to Reliquary I/Dock to Hillside Vista
   > Dock to Grand Court
-      Grapple Swing or Use Screw Attack (Space Jump)
+      All of the following:
+          After Hillside Vista Statue
+          Grapple Swing or Use Screw Attack (Space Jump)
 
 > Pickup (Missile Expansion); Heals? False
   * Pickup 28; Major Location? False
-  > Dock to Reliquary I
+  > Dock to Grand Court
+      Trivial
+
+> Event - Statue; Heals? False
+  * Event Hillside Vista Statue
+  > Dock to Grand Court
```

Comment:
Connect the event to Reliquary I because you are right in front of it after removing the statue